blogs/ecs-canary-deployments-pipeline/Makefile

```diff
@@ -0,0 +1,30 @@
+set-env:
+	@chmod +x ./setup/scripts/export_environment_variables.sh
```

Comment:
you can make these files executable ahead of time and git will honor that.

Reply:
Made them executable ahead of time. Fixed it by removing references from Makefile